addons/mail/static/src/messaging/entity/messaging_initializer/messaging_initializer.js

```diff
@@ -164,29 +153,30 @@ function MessagingInitializerFactory({ Entity }) {
          * @param {Object} param0.channel_slots
          * @param {Object[]} [param0.channel_slots.channel_channel=[]]
          * @param {Object[]} [param0.channel_slots.channel_direct_message=[]]
-         * @param {Object[]} [param0.channel_slots.channel_livechat=[]]
          * @param {Object[]} [param0.channel_slots.channel_private_group=[]]
          * @param {integer[]} [param0.moderation_channel_ids=[]]
          */
         _initChannels({
             channel_slots: {
                 channel_channel = [],
                 channel_direct_message = [],
-                // AKU FIXME: should be patch in im_livechat
-                channel_livechat = [],
                 channel_private_group = [],
             },
             moderation_channel_ids = [],
         }) {
-            for (const data of channel_channel.concat(channel_direct_message, channel_livechat, channel_private_group)) {
-                let channel = this.env.entities.Thread.channelFromId(data.id);
-                if (!channel) {
-                    channel = this.env.entities.Thread.create(
-                        Object.assign({}, data, { isPinned: true })
-                    );
-                }
-            }
+            // AKU FIXME: before any creation of channel/thread because this is
```

Comment:
fixme?

Reply:
ok, just added an attribute "moderatedChannelIds" on partner, and now relates to currentPartner's moderated channel ids. Also took this opportunity to rename "isModeratedByUser" to "isModeratedByCurrentPartner".